src/states/SelectAttackState.py

```diff
@@ -106,6 +106,7 @@ def update(self, dt, events):
                         if self.selectAttackTile>=0 and self.effect.maxRange>0:
                             if self.field[self.avilableAttackTile[self.selectAttackTile]].is_occupied():
                                 self.player.ChangeAnimation("multi_attack")
+                                self.enemy.ChangeAnimation("death")
```

Comment:
I think "death" animation should be use when the enemy is actually dead. If we don't have the damage animation, maybe go with blinking in red instead. (might be future improvement)

Reply:
Maybe further improvement